src/bin/cargo/commands/add.rs

```diff
@@ -67,14 +67,18 @@ The package will be removed from your features.")
                 .conflicts_with("build")
                 .long_help("Mark the dependency as public
 
-The dependency can be referenced in your library's public API."),
+The dependency can be referenced in your library's public API.
+
+Unstable (nightly-only) <https://doc.rust-lang.org/cargo/reference/unstable.html#public-dependency>"),
```

Comment:
The short help should have a trailing ` (unstable)`

Personally, I wouldn't add more than that to the long help.  Being an unstable feature, we likely don't want to draw more attention to it which an extra long help section would do

Reply:
> The short help should have a trailing  (unstable)
I think it's better this way.

Reply:
Thanks for you remind. 

(I totaly misunderstand the `short help`)

Reply:
Is anything need to tweak?

The finall help output is:

- cargo add --help 
```
      --public
          Mark the dependency as public (unstable)
          
          The dependency can be referenced in your library's public API.

      --no-public
          Mark the dependency as private (unstable)
          
          While you can use the crate in your implementation, it cannot be referenced in your public API.
```

- cargo help add

```
       --public
           Mark the dependency as public.

           Unstable (nightly-only) <https://doc.rust-lang.org/cargo/reference/unstable.html#public-dependency>

       --no-public
           Mark the dependency as private.

           Unstable (nightly-only) <https://doc.rust-lang.org/cargo/reference/unstable.html#public-dependency>

```